### PR TITLE
Update installation options on pulpproject.org

### PR DESCRIPTION
**Pull request by melcorr**
* Remove insta-demo
* Rewrite parts of intro page
* Add docker compose link
* Add podman compose docs

### CODE REVIEW

**Comment by melcorr** [comment]
Preview here: 

https://melcorr.github.io/pulpproject.org/installation-introduction/
https://melcorr.github.io/pulpproject.org/podman-compose/
https://melcorr.github.io/pulpproject.org/pulp-operator/

File: podman-compose.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Many members of the community have requested a production-ready container to deploy Pulp.
+In an effort to address this feedback, we have re-used images from [Pulp Operator](/pulp-operator/) to provide a podman compose.
+Because this podman compose is new, so far it has received limited feedback and testing.
+We want to hear from you. Let us know what you think on our [community discourse](https://discourse.pulpproject.org/).
+
+To use Pulp's podman compose, you must have an understanding of podman compose and also how to configure Pulp's podman compose to best suit your deployment requirements.
+At the moment, there is minimal documentation for Pulp's podman compose.
+If you'd like to add some tips, tricks, or tutorials, feel free to raise a PR against [this page](https://github.com/pulp/pulpproject.org/).
+
+Pulp's podman compose requires requires podman 3 or higher, as well as podman-compose 0.1 or higher.
```
Comment (fao89) [inline podman-compose.md]: Actually, it should be 3.x and 0.1.z (higher versions break it)
https://github.com/pulp/pulp-operator/tree/main/containers#compose
and you can point users to this link ^

Comment (melcorr) [reply podman-compose.md]: Interesting. I completely misread your statement! Thanks.

Comment (mdellweg) [reply podman-compose.md]:
```suggestion
Pulp's podman compose requires podman 3 or higher, as well as podman-compose 0.1 or higher.
```

Comment (melcorr) [reply podman-compose.md]: Fixed x 2

**Comment by melcorr** [comment]
Thanks, gents. 
If you have any other comments, let me know.

File: podman-compose.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+In an effort to address this feedback, we have re-used images from [Pulp Operator](/pulp-operator/) to provide a podman compose.
+Because this podman compose is new, so far it has received limited feedback and testing.
+We want to hear from you. Let us know what you think on our [community discourse](https://discourse.pulpproject.org/).
+
+To use Pulp's podman compose, you must have an understanding of podman compose and also how to configure Pulp's podman compose to best suit your deployment requirements.
+At the moment, there is minimal documentation for Pulp's podman compose.
+If you'd like to add some tips, tricks, or tutorials, feel free to raise a PR against [this page](https://github.com/pulp/pulpproject.org/).
+
+Pulp's podman compose is limited to versions podman 3.x and podman-compose 0.1.x.
+It does not work with higher versions.
```
Comment (fao89) [inline podman-compose.md:18]: 👍🏽

**Review by fao89 — APPROVED** [review]
LGTM

**Pull request merged**